src/tz_windows.rs

```diff
@@ -1,312 +1,13 @@
-use std::mem::zeroed;
+use windows::Globalization::Calendar;
 
-use windows_sys::core::{GUID, HRESULT};
-use windows_sys::Win32::System::Com::CoIncrementMTAUsage;
-use windows_sys::Win32::System::WinRT::HSTRING_HEADER;
-
-use self::hstring::HString;
-use self::instance::Instance;
-use self::tz_on_calendar::TzOnCalendar;
-
-macro_rules! wstring {
-    ($($letters:tt)+) => {
-        [ $($letters as _,)+ ]
-    };
-}
-
-const WINDOWS_GLOBALIZATION_CALENDAR: &[u16] = &wstring!(
-    'W' 'i' 'n' 'd' 'o' 'w' 's' '.'
-    'G' 'l' 'o' 'b' 'a' 'l' 'i' 'z' 'a' 't' 'i' 'o' 'n' '.'
-    'C' 'a' 'l' 'e' 'n' 'd' 'a' 'r'
-    0
-);
-
-const TIMEZONE_ON_CALENDAR_GUID: GUID = GUID {
-    data1: 0xbb3c25e5,
-    data2: 0x46cf,
-    data3: 0x4317,
-    data4: [0xa3, 0xf5, 0x02, 0x62, 0x1a, 0xd5, 0x44, 0x78],
-};
-
-const CO_E_NOTINITIALIZED: HRESULT = -2147221008;
-
-impl From<HRESULT> for crate::GetTimezoneError {
-    fn from(orig: HRESULT) -> Self {
-        std::io::Error::from_raw_os_error(orig).into()
+impl From<windows::core::Error> for crate::GetTimezoneError {
+    fn from(_orig: windows::core::Error) -> Self {
+        crate::GetTimezoneError::OsError
```

Comment:
This line could be made ↓ so the error message does not get lost:

```rust
GetTimezoneError::IoError(io::Error::new(io::ErrorKind::Other, orig))
```

Reply:
Nice. I put this in 1b898d1237192b54e831ba56e7e0b88152480e97.